py/examples/table_events_group.py

```diff
@@ -0,0 +1,114 @@
+# Table / Events / Group
+# Register the `group_change` #event to emit Wave event when group collapses or opens.
+# #table #events #groups #background_tasks
+# ---
+import asyncio
+import concurrent.futures
+import time
+from h2o_wave import main, app, Q, ui
+
+bobrows = [
+    {"name":"row1", "cell":"Issue1"},
+    {"name":"row2", "cell":"Issue2"},
+]
+johnrows = [
+    {"name":"row3", "cell":"Issue3"},
+    {"name":"row4", "cell":"Issue4"},
+    {"name":"row5", "cell":"Issue5"},
+]
+issue_cnt = 5
+
+collapsed_states = {
+    'Bob': True,
+    'John': False
+}
+stop = False
+new_issues_label = 'Add Issues'
+
+def add_issues_function(q: Q, loop: asyncio.AbstractEventLoop):
+    global stop
+    stop = False
+    future = None
+    while not stop:
+        time.sleep(2)
+        if not future or future.done():
+            future = asyncio.ensure_future(update_issues(q), loop=loop)
+
+async def update_issues(q: Q):
+    global issue_cnt
+    issue_cnt += 1
+    if (issue_cnt % 2) == 0:
+        bobrows.append({"name":"row"+str(issue_cnt), "cell":"Issue"+str(issue_cnt)})
+    else:
+        johnrows.append({"name":"row"+str(issue_cnt), "cell":"Issue"+str(issue_cnt)})   
+    update_table_groups(q)
+    await q.page.save()       
+
+
+@app('/demo')
+async def serve(q: Q):
```

Comment:
Love this example, but it might be too complex. Let's strip it down to a bare minimum - registering group change event and displaying the currently changed groups (ie without background tasks adding new rows).

Reply:
Yeah it is a complex example, but I was trying to replicate the original [discussion](https://github.com/h2oai/wave/discussions/2131) from when this ticket was originally created. It was certainly helpful for me when testing.

I've remove the bits related to adding new rows and updated the screenshot under ./website/docs/examples/assets.